docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,71 @@
+Canonical URLs
+==============
+
+A `Canonical URL`_
+allows you to specify the preferred version of a web page to prevent duplicated content.
+They are mainly used by search engines to link users to the correct
+version and domain of your documentation.
+
+If canonical URL's aren't used,
+it's easy for outdated documentation to be the top search result for various pages in your documentation.
+This is not a perfect solution for this problem,
+but generally people finding outdated documentation is a big problem,
+and this is one of the suggested ways to solve it from search engines.
+
+.. _Canonical URL: https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls
+
+How Read the Docs generates canonical URLs
+------------------------------------------
+
+The canonical URL takes into account:
+
+* The default version of your project (usually "latest" or "stable").
+* The canonical :doc:`custom domain </custom-domains>` if you have one,
+  otherwise the default :ref:`subdomain <hosting:subdomain support>` will be used.
+
+For example, if you have a project named ``example-docs``
+with a custom domain ``https://docs.example.com``.
+A search engine like Google will index both domains, ``https://example-docs.readthedocs.io`` and
+``https://docs.example.com``.
```

Comment:
This was a little fragmenty, but readers might not make the connection between `example-docs` and it's hosted domain `example-docs.readthedocs.io`

```suggestion
For example, if you have a project named ``example-docs``
with a custom domain ``https://docs.example.com``,
then your documentation will be served at ``https://example-docs.readthedocs.io`` and ``https://docs.example.com``.
Without specifying a canonical URL, a search engine like Google will index both domains.
```